sotodlib/site_pipeline/preprocess_tod.py

```diff
@@ -241,6 +270,18 @@ def get_parser(parser=None):
         help="Number of days (unit is days) in the past to start observation list.",
         type=int
     )
+    parser.add_argument(
+        '--write-block',
+        help="How many obs before writing to db.",
+        type=int,
+        default=10
+    )
+    parser.add_argument(
+        '--nthreads',
+        help="Number of parallel threads to run on.",
```

Comment:
multiprocessing uses parallel processes instead of parallel threads to avoid GIL